code/modules/projectiles/ammo_datums.dm

```diff
@@ -2079,7 +2078,7 @@ datum/ammo/bullet/revolver/tp44
 	name = "flare"
 	ping = null //no bounce off.
 	damage_type = BURN
-	flags_ammo_behavior = AMMO_INCENDIARY
+	flags_ammo_behavior = AMMO_EXPLOSIVE|AMMO_INCENDIARY
```

Comment:
???

Reply:
Ehh wierd i never changed that, its from a fresh master